.github/workflows/lint-pr.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Lint PR'
+name: 'Lint PR Title'
```

Comment:
Why this change?

Reply:
Workflow is only linting the PR title, not the entire PR (including contents), so updated it for clarity.